From 0658d3991572680777bfd00ac6971c9c93bec8c8 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Fri, 27 Feb 2015 15:06:35 +0100
Subject: [LIBREPORT PATCH] RHTSupport: attach reported_to when reporting from
 GUI

We want to provide GSS with an URL to ABRT server.

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/report_RHTSupport.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/report_RHTSupport.xml.in b/src/plugins/report_RHTSupport.xml.in
index b4e68f2..416af7c 100644
--- a/src/plugins/report_RHTSupport.xml.in
+++ b/src/plugins/report_RHTSupport.xml.in
@@ -4,7 +4,7 @@
     <_description>Create new Red Hat Support case</_description>
 
     <requires-items>package</requires-items>
-    <exclude-items-by-default>count,event_log,reported_to,vmcore</exclude-items-by-default>
+    <exclude-items-by-default>count,event_log,vmcore</exclude-items-by-default>
     <exclude-items-always></exclude-items-always>
     <exclude-binary-items>no</exclude-binary-items>
     <include-items-by-default></include-items-by-default>
-- 
1.8.3.1

